From 014394f10a11a1b36f8a2e8a0722ec2fddf74b40 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Caol=C3=A1n=20McNamara?= <caolanm@redhat.com>
Date: Fri, 9 Dec 2016 15:26:59 +0000
Subject: oss-fuzz we don't want this stream to automatically resize on seeks
 past end

Change-Id: Ie42b2e173cd3ed9faaf17c3c978227613129fa84
(cherry picked from commit f212bd45a7c7d97fb16380cd9fa7c6abbfb71e05)
(cherry picked from commit b832d7019f7e7499f5fb3da5d5f49ddbf0f35585)
Reviewed-on: https://gerrit.libreoffice.org/31803
Tested-by: Jenkins <ci@libreoffice.org>
Reviewed-by: Michael Stahl <mstahl@redhat.com>

diff --git a/vcl/source/filter/wmf/winwmf.cxx b/vcl/source/filter/wmf/winwmf.cxx
index 88ab38f..c4f7e78 100644
--- a/vcl/source/filter/wmf/winwmf.cxx
+++ b/vcl/source/filter/wmf/winwmf.cxx
@@ -1093,7 +1093,7 @@ void WMFReader::ReadRecordParams( sal_uInt16 nFunc )
                             {   // first EMF comment
                                 nEMFRecCount    = nComRecCount;
                                 nEMFSize        = nEMFTotalSize;
-                                pEMFStream = o3tl::make_unique<SvMemoryStream>( nEMFSize );
+                                pEMFStream = o3tl::make_unique<SvMemoryStream>(nEMFSize, 0);
                             }
                             else if( ( nEMFRecCount != nComRecCount ) || ( nEMFSize != nEMFTotalSize ) ) // add additional checks here
                             {
-- 
cgit v0.10.2


